package.json

```diff
@@ -1080,6 +1090,16 @@
                                 "pattern": "^(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])\\.(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])\\.(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])\\.(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])$",
                                 "type": "string"
                             },
+                            "portRangeStart": {
+                                "default": null,
+                                "description": "Start range for GDB port scan. Optional.",
+                                "type": "string"
+                            },
+                            "portRangeEnd": {
+                                "default": null,
```

Comment:
Set it to `-1`.

Reply:
I don't think a portRangeEnd is needed at all. Extension scans for 2000 ports starting from "portRangeStart" and we are done. If if you want to have a "portRangeEnd", you may want to specify as "numPortsToScan", the default being 2000 like it is now. and it better be a positive integer. Some targets need a minimum set of targets, so having this property creates additional error checking to make sure we satisfy the max. 2000 is a huge set.